### ISSUE TITLE
Problem with custom_error_response in version 3.0.2

### ISSUE THREAD

**Issue by zahorniak** [terraform-aws-modules/terraform-aws-cloudfront#90]
## Description

Hey. After upgrading to version [3.0.2](https://github.com/terraform-aws-modules/terraform-aws-cloudfront/pull/89) we started to get the following error:

```hcl
Error: Invalid index
on .terraform/modules/cdn/main.tf line 252, in resource "aws_cloudfront_distribution" "this":
      error_code = custom_error_response.value["error_code"]
custom_error_response.value is an object with no attributes
The given key does not identify an element in this collection value.
```

We're not using the variable `custom_error_response` at all. 

- [x] ✋ I have searched the open/closed issues and my issue is not listed.

## Versions

- Module version [Required]:
`3.0.2`

- Terraform version:
`1.3.4`

- Provider version(s):

```
+ provider registry.terraform.io/hashicorp/archive v2.2.0
+ provider registry.terraform.io/hashicorp/aws v4.39.0
+ provider registry.terraform.io/hashicorp/cloudinit v2.2.0
+ provider registry.terraform.io/hashicorp/external v2.2.3
+ provider registry.terraform.io/hashicorp/local v2.2.3
+ provider registry.terraform.io/hashicorp/null v3.2.0
+ provider registry.terraform.io/hashicorp/random v3.4.3
```

## Reproduction Code [Required]

```hcl
module "cdn" {
  source  = "registry.terraform.io/terraform-aws-modules/cloudfront/aws"
  version = "~> 3.0"

  aliases                        = [local.cdn_domain_name]
  comment                        = "${var.environment} File CDN"
  enabled                        = true
  is_ipv6_enabled                = true
  price_class                    = "PriceClass_100"
  retain_on_delete               = false
  wait_for_deployment            = false

  origin = {
    website_origin = {
      domain_name = local.backend_domain_name
      custom_origin_config = {
        http_port              = 80
        https_port             = 443
        origin_protocol_policy = "match-viewer"
        origin_ssl_protocols   = ["TLSv1", "TLSv1.1", "TLSv1.2"]
      }
    }
  }

  default_cache_behavior = {
    target_origin_id       = "website_origin"
    viewer_protocol_policy = "redirect-to-https"

    allowed_methods = ["GET", "HEAD", "OPTIONS", "PUT", "POST", "DELETE", "PATCH"]
    cached_methods  = ["GET", "HEAD"]
    compress        = true

    min_ttl     = 3600
    max_ttl     = 31536000
    default_ttl = 86400

    use_forwarded_values = true
  }

  viewer_certificate = {
    acm_certificate_arn      = module.acm_cdn.acm_certificate_arn
    ssl_support_method       = "sni-only"
    minimum_protocol_version = "TLSv1.2_2021"
  }

}
```

## Steps to reproduce the behavior:
Run `terraform plan`

## Expected behavior

The plan should not fail

## Actual behavior

Plan fails

### Terminal Output Screenshot(s)

![CleanShot 2022-11-14 at 20 23 03](https://user-images.githubusercontent.com/7808206/201736984-49ea4625-dfa1-4261-a13f-4bd9c006d0af.png)

## Additional context

We're using Terraform Cloud and not local machines. So the problem is not related to cache, etc.

**Comment by bin3377** [comment]
same here. new error pop up this week

**Comment by bin3377** [comment]
should be caused by this fix - https://github.com/terraform-aws-modules/terraform-aws-cloudfront/pull/89
no `custom_error_response` will cause problem

**Comment by antonbabenko** [comment]
This issue has been resolved in version 3.0.3 :tada:

**Comment by antonbabenko** [comment]
Fixed. Apologies for the bug! I thought about one and multiple items but forgot to check a case with no items.
